crates/k8s-version/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Bump rust-toolchain to 1.79.0 ([#xxx])
```

Comment:
note: This link needs updating in all changelogs.

Reply:
Oh yeah, this PR wasn't ready for review yet. Pretty sure I left it in In Progress.

But will leave this here so I definitely won't forget.

Reply:
Oh yeah, I already suspected that. Sorry for the premature review.

Reply:
All good. Just a few changes to come in, then we're good to go.

Reply:
Done in 196a44f